src/__tests__/scenes/TransactionDetailsScene.test.tsx

```diff
@@ -54,7 +54,8 @@ describe('TransactionDetailsScene', () => {
     core: {
       account: {
         currencyWallets: { '123': fakeCoreWallet },
-        currencyConfig: { bitcoin: fakeCurrencyConfig }
+        currencyConfig: { bitcoin: fakeCurrencyConfig },
+        watch() {}
```

Comment:
Aha! This is why I never implemented `useWatch` in the original version of the hook. I knew there was a reason, but I just couldn't remember.